azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/middleware/sdk/ChatActionHandler.kt

```diff
@@ -102,6 +102,12 @@ internal class ChatActionHandler(private val chatService: ChatService) {
             val error = ChatStateError(errorCode = ErrorCode.CHAT_START_EVENT_NOTIFICATIONS_FAILED)
             dispatch(ErrorAction.ChatStateErrorOccurred(chatStateError = error))
         }
+        try {
+            chatService.requestChatParticipants()
+        } catch (ex: Exception) {
+            val error = ChatStateError(errorCode = ErrorCode.CHAT_REQUEST_PARTICIPANTS_FAILED)
```

Comment:
can we say CHAT_REQUEST_PARTICIPANTS_FETCH_FAILED?

Reply:
Renamed in https://github.com/Azure/communication-ui-library-android/pull/494/commits/f7c9186588ba0c2dba29a4b50ae3f9b14c532b34